strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/test/java/org/carlspring/strongbox/config/NpmLayoutProviderTestConfig.java

```diff
@@ -14,6 +15,7 @@
           EventsConfig.class,
           DataServiceConfig.class,
           CommonConfig.class,
+          CronTasksConfig.class,
```

Comment:
Why is this necessary?

Reply:
Could not start without it. It's already added in other configurations, for example in `Maven2LayoutProviderTestConfig`. I don't have enough knowledge about whole project to answer with details. :(

Reply:
@fuss86 / @sbespalov : Is this required? Any advice?

Reply:
In `NpmRepositoryManagementStrategy`:
```
@Inject
private CronTaskDataService cronTaskDataService;
```
Without `CronTasksConfig` I am receiving `NoSuchBeanDefinitionException`.

Reply:
@ptirador @fuss86 @sbespalov Could you guys help with that?

Reply:
Yeah, it looks like `NpmLayoutProviderTestConfig` is not used nowhere else except `NpmLayoutProviderCronTasksTestConfig`. Because, as it looks like, we still need `CronTasksConfig` in tests from outside of the `cron tasks tests` I suggest to merge `@Import`s into `NpmLayoutProviderTestConfig` and use only `NpmLayoutProviderTestConfig` and drop `NpmLayoutProviderCronTasksTestConfig`

Reply:
`NpmLayoutProviderCronTasksTestConfig` removed.